src/libecalc/core/graph_result.py

```diff
@@ -649,13 +770,20 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
                                 is_valid=TimeSeriesBoolean(
                                     timesteps=model.timesteps, values=model.is_valid, unit=Unit.NONE
                                 ),
-                                energy_usage_cumulative=model.energy_usage.to_volumes().cumulative(),
+                                energy_usage_cumulative=model.energy_usage.to_timeseries_rate(
+                                    regularity=regularity.values
+                                )
+                                .to_volumes()
+                                .cumulative(),
                                 power_cumulative=(
-                                    model.power.to_volumes().to_unit(Unit.GIGA_WATT_HOURS).cumulative()
+                                    model.power.to_timeseries_rate(regularity=regularity.values)
+                                    .to_volumes()
+                                    .to_unit(Unit.GIGA_WATT_HOURS)
+                                    .cumulative()
                                     if model.power is not None
                                     else None
                                 ),
-                                power=model.power,
+                                power=model.power.to_timeseries_rate(regularity=regularity.values),
```

Comment:
power can be None, so:
```suggestion
power=model.power.to_timeseries_rate(regularity=regularity.values) if model.power is not None else None,
```